### ISSUE TITLE
Unable to Install .rpm installation file on Fedora 39

### ISSUE THREAD

**Issue by dwtoledo** [AppFlowy-IO/AppFlowy#4400]
I'm not able to install the latest AppFlowy .rpm (version 0.4.2) on Fedora 39 (GNOME 45).
When I open .rpm file using Fedora Software app and click on install button it returns the following error "Unable to install AppFlowy" log:

"Could not depsolve transaction; 1 problem detected:
 Problem: conflicting requests
  - nothing provides libfile_selector_linux_plugin.so()(64bit) needed by AppFlowy-0.4.2-2.x86_64 from @commandline
  - nothing provides libflowy_infra_ui_plugin.so()(64bit) needed by AppFlowy-0.4.2-2.x86_64 from @commandline
  - nothing provides libflutter_linux_gtk.so()(64bit) needed by AppFlowy-0.4.2-2.x86_64 from @commandline
  - nothing provides libgtk_plugin.so()(64bit) needed by AppFlowy-0.4.2-2.x86_64 from @commandline
  - nothing provides libhotkey_manager_plugin.so()(64bit) needed by AppFlowy-0.4.2-2.x86_64 from @commandline
  - nothing provides libirondash_engine_context_plugin.so()(64bit) needed by AppFlowy-0.4.2-2.x86_64 from @commandline
  - nothing provides liblocal_notifier_plugin.so()(64bit) needed by AppFlowy-0.4.2-2.x86_64 from @commandline
  - nothing provides librich_clipboard_linux_plugin.so()(64bit) needed by AppFlowy-0.4.2-2.x86_64 from @commandline
  - nothing provides libscreen_retriever_plugin.so()(64bit) needed by AppFlowy-0.4.2-2.x86_64 from @commandline
  - nothing provides libsuper_native_extensions_plugin.so()(64bit) needed by AppFlowy-0.4.2-2.x86_64 from @commandline
  - nothing provides liburl_launcher_linux_plugin.so()(64bit) needed by AppFlowy-0.4.2-2.x86_64 from @commandline
  - nothing provides libwindow_manager_plugin.so()(64bit) needed by AppFlowy-0.4.2-2.x86_64 from @commandline"

I also tried install the previous previous version 0.4.1, same problem.
I'm able to install only version 0.4.0 using Fedora Software app instead of download .rpm file.

**Comment by Cyberphinx** [comment]
I'm having the exact same problem with fedora 39 with  sudo dnf install AppFlowy-0.5.3-linux-x86_64.rpm

**Comment by Bug-g0** [comment]
same here with fedora 40 with sudo dnf install AppFlowy-0.6.2-linux-x86_64.rpm

**Comment by MohitSG96** [comment]
same error here
Could not depsolve transaction; 1 problem detected:
 Problem: conflicting requests
  - nothing provides libbitsdojo_window_linux_plugin.so()(64bit) needed by AppFlowy-0.6.7.2-2.x86_64 from @commandline
  - nothing provides libdesktop_drop_plugin.so()(64bit) needed by AppFlowy-0.6.7.2-2.x86_64 from @commandline
  - nothing provides libfile_selector_linux_plugin.so()(64bit) needed by AppFlowy-0.6.7.2-2.x86_64 from @commandline
  - nothing provides libflowy_infra_ui_plugin.so()(64bit) needed by AppFlowy-0.6.7.2-2.x86_64 from @commandline
  - nothing provides libflutter_linux_gtk.so()(64bit) needed by AppFlowy-0.6.7.2-2.x86_64 from @commandline
  - nothing provides libgtk_plugin.so()(64bit) needed by AppFlowy-0.6.7.2-2.x86_64 from @commandline
  - nothing provides libhotkey_manager_plugin.so()(64bit) needed by AppFlowy-0.6.7.2-2.x86_64 from @commandline
  - nothing provides libirondash_engine_context_plugin.so()(64bit) needed by AppFlowy-0.6.7.2-2.x86_64 from @commandline
  - nothing provides liblocal_notifier_plugin.so()(64bit) needed by AppFlowy-0.6.7.2-2.x86_64 from @commandline
  - nothing provides libscreen_retriever_plugin.so()(64bit) needed by AppFlowy-0.6.7.2-2.x86_64 from @commandline
  - nothing provides libsuper_native_extensions_plugin.so()(64bit) needed by AppFlowy-0.6.7.2-2.x86_64 from @commandline
  - nothing provides liburl_launcher_linux_plugin.so()(64bit) needed by AppFlowy-0.6.7.2-2.x86_64 from @commandline
  - nothing provides libwindow_manager_plugin.so()(64bit) needed by AppFlowy-0.6.7.2-2.x86_64 from @commandline

**Comment by ahmedsomaa** [comment]
Same here on Fedora 41. Any solution?

**Comment by mkaramuk** [comment]
I just moved to Fedora 41 and tried to install AppFlowy RPM package and bumped into the same issue. I don't know too much about Fedora package management but I wonder are those dependencies come from some 3rd party repositories such as RPMFusion?

**Comment by khorshuheng** [comment]
No, it's not third party dependency. Our rpm package is indeed broken at the moment as the rpm package is converted from deb to rpm via the alien tool, and it didn't work as expected. We will likely need a dedicated CI workflow for RPM package exclusively.

For the time being, please use flatpak/AppImage for Fedora until we fixed the build process.

**Comment by mkaramuk** [comment]
> No, it's not third party dependency. Our rpm package is indeed broken at the moment as the rpm package is converted from deb to rpm via the alien tool, and it didn't work as expected. We will likely need a dedicated CI workflow for RPM package exclusively.
> 
> For the time being, please use flatpak/AppImage for Fedora until we fixed the build process.

Hey @khorshuheng, thanks for the clarification. I'll try that.

**Comment by LucasXu0** [comment]
Follow the Linux installation issues on https://github.com/AppFlowy-IO/AppFlowy/issues/8854
